src/js/common/schemaform/review/SubmitButtons.jsx

```diff
@@ -29,6 +29,21 @@ export default function SubmitButtons({ submission, onSubmit, onBack, errorMessa
           buttonClass="form-button-green"
           beforeText="&#10003;"/>
     );
+  } else if (submission.status === 'clientError') {
+    submitButton = (
+      <ProgressButton
+          onButtonClick={onSubmit}
+          buttonText="Submit Application"
+          buttonClass="usa-button-primary"/>
+    );
+    submitMessage = (
```

Comment:
Not sure that abstracting this out makes much sense, but could we reference these duplicate blocks in a comment in case we have to update the text?